e2e/tests/session_steps_test.go

```diff
@@ -86,17 +85,21 @@ func (s *suite) AfterTheSupplierCreatesAClaimForTheSessionForServiceForApplicati
 func (s *suite) TheClaimCreatedBySupplierForServiceForApplicationShouldBePersistedOnchain(supplierName, serviceId, appName string) {
 	ctx := context.Background()
 
-	claimsRes, err := s.supplierQueryClient.AllClaims(ctx, &suppliertypes.QueryAllClaimsRequest{
+	allClaimsRes, err := s.supplierQueryClient.AllClaims(ctx, &suppliertypes.QueryAllClaimsRequest{
 		Filter: &suppliertypes.QueryAllClaimsRequest_SupplierAddress{
 			SupplierAddress: accNameToAddrMap[supplierName],
 		},
 	})
 	require.NoError(s, err)
-	require.NotNil(s, claimsRes)
+	require.NotNil(s, allClaimsRes)
 
 	// Assert that the number of claims has increased by one.
 	preExistingClaims := s.scenarioState[preExistingClaimsKey].([]suppliertypes.Claim)
-	require.Len(s, claimsRes.Claim, len(preExistingClaims)+1)
+	// NB: We are avoiding the use of require.Len here because it provides unreadable output
+	// TODO_TECHDEBT: Due to the speed of the blocks of the LocalNet sequencer, along with the small number
+	// of blocks per session, multiple claims may be created throughout the duration of the test. Until
+	// these values are appropriately adjusted
+	require.Greater(s, len(allClaimsRes.Claim), len(preExistingClaims), "number of claims must have increased")
```

Comment:
I think `require.Greater()` works better here in general. :100: 

I'm not so convinced that the comment against usage of `require.Len()`  is warranted, did you experiment with `require.Lenf()`?

Reply:
> I think require.Greater() works better here in general and the part of the comment explaining this assertion is 👌. 💯

👍 

> I'm less convinced that the part of the comment against usage of require.Len() is warranted, did you experiment with require.Lenf()?

I did experiment with `Lenf`, but I did experiment with giving `require.Len` an unformatted string.

When there's an error, it prints out the entire claims slice. This was an array of serialized protos and large in length, which looked incomprehensible on the screen and confusing to the reader.

Reply:
I did experiment with Lenf, but I did experiment with giving require.Len an unformatted string.

I'm assuming one of those is suppose to be a "didn't". :sweat_smile: 

---

For the record, in other  cases, I would argue that `#Lenf()` produces the least ambiguous output. I agree that the output can be large if the list is not empty; however, that information is useful when debugging, and isn't obscuring the reason for failure as it's on its own line.

### `#Lenf()`
```go
func TestSanity(t *testing.T) {
	x := []struct{ a, b int }{{1, 2}, {3, 4}}
	require.Lenf(t, x, 3, "x should have 3 elements, got %d", len(x))
}
```
![image](https://github.com/pokt-network/poktroll/assets/600733/cad80f5c-48ef-4522-812e-ac87f5e3be7f)

### `#Len()`
```go
func TestSanity(t *testing.T) {
	x := []struct{ a, b int }{{1, 2}, {3, 4}}
	require.Len(t, x, 3)
}
```
![image](https://github.com/pokt-network/poktroll/assets/600733/2b9ec6ae-e5e0-406a-adbf-0cd13d7e48f3)

### `#Greater()`
```go
func TestSanity(t *testing.T) {
	x := []struct{ a, b int }{{1, 2}, {3, 4}}
	require.Greater(t, len(x), 2)
}
```
![image](https://github.com/pokt-network/poktroll/assets/600733/7c7634b2-6830-4245-afb9-808492b17c6f)